src/assets/data/edition/series/2/section/2a/m36/source-list.json

```diff
@@ -3,15 +3,15 @@
         {
             "siglum": "A",
             "siglumAddendum": "",
-            "type": "Skizzen zu <br /><span class=\"tab\"></span>Studienkomposition für Klavier M 36.<br />Enthält auch Skizzen zu Studienkomposition für Klavier M 34–35, M 37–38, M 39 Textfassung 1→2 und M* 404.",
+            "type": "Skizzen zu <br /><span class=\"tab\"></span>Studienkomposition für Klavier M 36.<br />Enthält auch Skizzen zu Studienkomposition für Klavier M 34, M 35/42, M 37–39 und M* 404.",
```

Comment:
Weglassen von "Textfassung 1->2" intendiert? Dann bei Quellenlisten von M34-M38 an dieser Stelle auch?

Reply:
Ja, intendiert. Und in dem Fall leider auch ja: bei M 34 bis M 38 anpasssen.